dask_sql/physical/rel/logical/filter.py

```diff
@@ -1,11 +1,13 @@
 import logging
 from typing import TYPE_CHECKING, Union
 
+import dask.config as dask_config
 import dask.dataframe as dd
 import numpy as np
 
 from dask_sql.datacontainer import DataContainer
 from dask_sql.physical.rel.base import BaseRelPlugin
+from dask_sql.physical.rel.logical.optimize import attempt_predicate_pushdown
```

Comment:
This is a minor nitpick, but since optimize.py doesn't actually contain any plugins to convert from SQL -> Dask, and at least for now I assume it is only going to be used within the scope of filter operations, I think it would make sense to relocate it to `dask_sql.physical.utils.filter` (though we could go for `optimize` if we expect these utils to be used in other places later on).

As an example, consider the relationship between:

- https://github.com/dask-contrib/dask-sql/blob/main/dask_sql/physical/rel/logical/sort.py
- https://github.com/dask-contrib/dask-sql/blob/main/dask_sql/physical/utils/sort.py

I'll also note that I'm really not sure if this is the best way to be doing things, but keeping it consistent for now should make it easier to refactor if/when the time comes 🙂